routes/members.js

```diff
@@ -284,4 +291,39 @@ router.patch("/delete-isMember", authenticate, authorizeUser("superUser"), membe
 
 router.patch("/archiveMembers/:username", authenticate, authorizeUser(SUPER_USER), members.archiveMembers);
 
+/**
+ * @swagger
+ * /members/add-default-archived-role:
+ *  patch:
+ *   summary: One time call to add default archived role for the users only where it does not exists
+ *   tags:
+ *     - Members
+ *   responses:
+ *     200:
+ *       description: Details of the users migrated
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/migratedUsers'
+ *     401:
+ *       description: unAuthorized
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/unAuthorized'
+ *     403:
+ *       description: forbidden
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/forbidden'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.patch("/add-default-archived-role", authenticate, authorizeUser("superUser"), members.addDefaultArchivedRole);
```

Comment:
The naming of API can be better, this is too long to have
These can help you 
https://www.thoughtworks.com/en-us/insights/blog/rest-api-design-resource-modeling
https://aws.amazon.com/what-is/restful-api/

Reply:
Ok. I'll give them a read. 
The points I considered for naming this api were :
1. It's intended to be used just once
2. Similar naming convention has been followed for other migration endpoints like delete-isMember and member-to-role-migration

Reply:
Ok, then I guess it's fine, thanks for clarifying